Anytype/Sources/Models/Extensions/ObjectDetailsExtensions.swift

```diff
@@ -152,7 +160,21 @@ extension BundledRelationsValueProvider {
         objectType.isTemplateType
     }
     
-    var canCreateObject: Bool {
-        setOf.first { $0.isNotEmpty } != nil || isCollection
+    func isGroupAndCanCreateObject(setDocument: SetDocumentProtocol) -> Bool {
```

Comment:
I prefere to provide setDocument instead of just isRelationsSet: Bool to make it simpler for new api users to understand where to even find this boolean